src/skins/vector/css/matrix-react-sdk/views/messages/MTextBody.css

```diff
@@ -13,12 +13,3 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-
-.mx_MTextBody {
-    white-space: pre-wrap;
```

Comment:
Isn't this what `pre` blocks are for? When pasting text in from somewhere, the white-space tends to seriously break the layout and make things ugly. What is the use case for this?